CVE-2021-32682/data/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '2'
+services:
+ web:
+   image: vulhub/elfinder:2.1.58
```

Comment:
Could you please download the Docker image, put it to the data folder and install it from file?